app/models/work_package/pdf_export/export/wp/attributes.rb

```diff
@@ -156,6 +160,10 @@ def group_attributes_group_parts(group, work_package)
     parts
   end
 
+  def skip_attribute?(form_key, work_package)
+    !CustomField.custom_field_attribute?(form_key) && !user_allowed_view_attribute?(work_package, form_key)
+  end
```

Comment:
```suggestion
  def show_attribute?(form_key, work_package)
    CustomField.custom_field_attribute?(form_key) || user_allowed_view_attribute?(work_package, form_key)
  end
```
or same as in other class `can_view_attribute?`